docker/README.md

```diff
@@ -32,6 +32,19 @@ copy the certificate to `nginx-plus-server/nginx-repo.crt` and the key to `nginx
 Another option is to change the paths to the certificate and key files in lines 12 and 13 of
 `nginx-plus-server/Dockerfile`.
 
+## Run with the newer NGINX Plus API
+
+Current `nginx-plus-server/Dockerfile` is configured to run the NGINX+ with the legacy APIs. In order to run
+the NGINX+ with the newer APIs replace line no. 6 to 13 with the following configuration in `nginx-plus-server/nginx.conf`:
+
+```nginx
+location /api {
```

Comment:
Should the base path be configurable in this plugin as well?  If users are required to add this directive and specify the path to expose the api I'd think so.

Reply:
Sorry, I am not getting it. which base path are you referring to?

Reply:
Sorry, I am not getting it. which base path are you talking about?

Reply:
here `/api` is being set to use the http api module, but it could be any base path like `/some/other/path`.  If users do this then the plugin isn't compatible with their environment and there's no workaround afaict.

Reply:
Hello @rmfitzpatrick,

As per the latest design(that exists in the current source code), we have added the config option of `APIBasePath`.
This facilitates the end-user to provide a configurable API base path. This config option is applicable for both the `legacy` and `newer` APIs. It goes hand-in-hand with the config option `APIVersion`. If `APIVersion` is set to `0`(which means `legacy` APIs) then, the respective base path for the `legacy` APIs should be provided(defaults to `/status`). If `APIVersion` is set to a positive integer value (which means `newer` APIs) (defaults to `3` if not provided) then, the respective base path for the `newer` APIs should be provided(defaults to `/api`).

Please let us know in case of any queries.